internal/collector/vector/visitors.go

```diff
@@ -33,17 +34,10 @@ func CollectorVisitor(collectorContainer *corev1.Container, podSpec *corev1.PodS
 	collectorContainer.Args = []string{entrypointValue}
 
 	podSpec.Volumes = append(podSpec.Volumes,
-		corev1.Volume{Name: common.ConfigVolumeName, VolumeSource: corev1.VolumeSource{Secret: &corev1.SecretVolumeSource{
-			SecretName: resNames.ConfigMap,
-			Items:      []corev1.KeyToPath{{Key: ConfigFile, Path: ConfigFile}},
-			Optional:   utils.GetPtr(true),
-		}}},
+		corev1.Volume{Name: common.ConfigVolumeName, VolumeSource: corev1.VolumeSource{ConfigMap: &corev1.ConfigMapVolumeSource{LocalObjectReference: corev1.LocalObjectReference{Name: resNames.ConfigMap}}}},
 		corev1.Volume{Name: common.DataDir, VolumeSource: corev1.VolumeSource{HostPath: &corev1.HostPathVolumeSource{Path: dataPath}}},
-		corev1.Volume{Name: common.EntrypointVolumeName, VolumeSource: corev1.VolumeSource{Secret: &corev1.SecretVolumeSource{
-			SecretName: resNames.ConfigMap,
-			Items:      []corev1.KeyToPath{{Key: RunVectorFile, Path: RunVectorFile}},
-			Optional:   utils.GetPtr(true),
-		}}},
+		corev1.Volume{Name: common.EntrypointVolumeName, VolumeSource: corev1.VolumeSource{ConfigMap: &corev1.ConfigMapVolumeSource{LocalObjectReference: corev1.LocalObjectReference{Name: resNames.ConfigMap}}}},
+		corev1.Volume{Name: common.SecretDataReader, VolumeSource: corev1.VolumeSource{ConfigMap: &corev1.ConfigMapVolumeSource{LocalObjectReference: corev1.LocalObjectReference{Name: resNames.ConfigMap}}}},
```

Comment:
This volumesource should be secret since the readerscript is full of secrets

Reply:
oh, yes exactly

Reply:
I see from your comment that we could leave this as a configmap since the secrets are obscured by files.  Given you have already completed the changes we should just leave it